fastlane/metadata/tr/subtitle.txt

```diff
@@ -1 +1 @@
-Bir web sitesi veya blog oluşturun
\ No newline at end of file
+Bir web sitesi oluşturun
```

Comment:
Manually updated this to get within the 30 chars limit. In 17.8, I'll make sure to add the comment regarding the length of the subtitle in the `.po` and ping i18n about it to see if they consider these translations appropriate.